Start implementing fixes for Python 3.10

benmoran56 opened the pull request:
I did some minimal fixes so that Samura-IDE launches again on Python 3.10.
Further work is definitely needed, but it's a start.

Poikilos commented:
Could you make the PEP8 style changes so we can see if other tests pass? I want to see if this works on the other Python versions but the other tests won't run unless this passes.

benmoran56 commented:
I fixed the PEP errors, and all the tests complete now. I also added Python 3.10 to the test matrix.

The existing tests are passing, but of course many things are probably still broken. It's a start!

Poikilos commented:
Thanks very much
